oslabs-beta/Kubiquity
# Add responsive styling

WHAT DOES THIS PR DO? 
1. Styles the Navbar
2. Adds responsive styling to the app
3. Additional app style changes. 

HOW TO TEST THIS PR:
1. Run: `gh pr checkout 26 && npm install && npm run dev`
2. Once the app loads, verify that the styling looks good-ish, mainly the Navbar. 
3. Resize the app below 850px width. Verify the styling changes and looks good. 
4. Resize the app below 600px width. Verify that the app has an x-scroll bar. 

ADDITIONAL NOTES: 
1. As Rob as noted, this current design is really Facebook-y. To be fixed in a future PR.
